bpfman/src/lib.rs

```diff
@@ -737,6 +762,105 @@ fn filter(
         })
 }
 
+fn get_txc_programs(root_db: &Db, if_index: u32, direction: Direction) -> Vec<TcxProgram> {
+    root_db
+        .tree_names()
+        .into_iter()
+        .filter(|p| bytes_to_string(p).contains(PROGRAM_PREFIX))
+        .filter_map(|p| {
+            let id = bytes_to_string(&p)
+                .split('_')
+                .last()
+                .unwrap()
+                .parse::<u32>()
+                .unwrap();
+            let tree = root_db.open_tree(p).expect("unable to open database tree");
```

Comment:
`.expect()` calls are scattered throughout the code, but if they are hit, they are a panic. I would like to avoid adding any new ones and slowly remove all the existing ones in the code.

Reply:
Done.